modules/ROOT/pages/push/admin-ui.adoc

```diff
@@ -0,0 +1,83 @@
+include::{partialsdir}/attributes.adoc[]
+
+include::./con_accessing-push-using-the-ui.adoc[leveloffset=+1]
+
+=== Managing registered device
+
+The details view of a specific variant shows a list of the registered devices at the bottom:
+
+image::ups-admin-devices1.png[Device list]
+
+The table shows the metadata of each device, including the device token, subscribed categories or - if present - the used alias. On this screen you could also exclude a specific device from receiving Push Notifications, by unchecking the _Receiving_ check-box.
```

Comment:
Suggestion: ... subscribed categories _and_ its alias, if present.

Reply:
Also, I think the preferred way is bold instead of italics in **Receiving**. @jstaffor @pb82 ?

Reply:
How about

The table shows the metadata for each device, including
* the device token
* subscribed categories
* the device alias, if an alias exists